config/routes.rb

```diff
@@ -1,2 +1,6 @@
 Rails.application.routes.draw do
-end
+  scope module: :api, defaults: { format: 'json' } do
+    namespace :v1 do
+    end
+    end
```

Comment:
Make sure your indentation is constant. We use two spaces (which you have) and every block stars and end on the same indent.